csxtools/utils.py

```diff
@@ -1,9 +1,6 @@
 import numpy as np
 import time as ttime
-try:
-    from databroker import get_images
-except ImportError:
-    from dataportal import get_images
+from databroker import get_images
```

Comment:
@ericdill Is there any harm in this, this was done because there are some old versions out there (on the jupyter hub) which have the old dataportal. Is this so old we should remove it?